src/plugins/intel_gpu/src/graph/impls/ocl/primitive_base.hpp

```diff
@@ -224,9 +224,9 @@ struct typed_primitive_impl_ocl : public typed_primitive_impl<PType> {
     }
 
     kernel_arguments_data get_arguments_impl(const typed_primitive_inst<PType>& instance) const override {
-        for (size_t k = 0; k < _kernels.size(); ++k) {
+        if (_kernels.size()) {
```

Comment:
Why?

Reply:
The loop makes only one step, if _kernels.size()>0. Return in line 235.

Reply:
@e-ddykim Is it expected behavior?

Reply:
It seems that yes.